## From the Debian package gcc-h8300-hms (patch 000_no_libstdc++-v3)
## DP: Description: Do not build libstdc++-v3 on h8300
## DP: Author: Michael Tautschnig <tautschn@model.in.tum.de>
## DP: Upstream status: N/A
## DP: Date: Tue, 07 Nov 2006 00:56:14 +0100

diff -ur gcc-h8300-hms-3.4.6/configure.in gcc-h8300-hms-3.4.6.patched/configure.in
--- gcc-h8300-hms-3.4.6.orig/configure.in
+++ gcc-h8300-hms-3.4.6/configure.in
@@ -155,7 +155,6 @@
 target_libraries="target-libiberty \
 		target-libgloss \
 		target-newlib \
-		target-libstdc++-v3 \
 		target-libf2c \
 		${libgcj} \
 		target-libobjc"
